### PR TITLE
Issue #1 Ability to move user

### PR DESCRIPTION
**Pull request by stephenlee91**
Summary: Initial commit for RasenBot. The bot is now able to move users who are sitting in voice channels with a single command. 

What & How: A defined move function that uses discord.py commands to move discord members specified in the single command sitting in discord voice channels and moves them to another voice channel that the user specifies in the single command. A token is taken from secrettoken.py to help hide the the actual token from the internet and is used to access RasenBot.

### CODE REVIEW

File: moveusers.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# async def activity_checker(member, id, activity, voicechannels, general_channel):
+#     for vc in voicechannels:
+#         if str(activity) == str(vc):
+#             print("here")
+#             await member.move_to(vc)
+ 
+ 
+ 
+ 
+
```
Comment (TimothyYLin) [inline moveusers.py]: Remove extraneous whitespace.

File: moveusers.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ 
+intents = discord.Intents.default()
+intents.presences = True
+intents.members = True
+ 
+client = commands.Bot(command_prefix='$', intents=intents)
+ 
+ 
+
+# @client.event
```
Comment (TimothyYLin) [inline moveusers.py:12]: Explain commented code and purpose briefly

File: moveusers.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ 
+    # for member in general_channel.members:
+    #     await activity_checker(member, member.id, member.activity, voicechannels, general_channel)
+ 
+# @client.commands
+# async def on_member_join(member):
+#     print(member)
+   
+ 
+# @client.command()
```
Comment (TimothyYLin) [inline moveusers.py:38]: Ditto

File: moveusers.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import discord
```
Comment (benjaminl4917) [inline moveusers.py:1]: Bro why does your code look exactly like mine

Comment (benjaminl4917) [reply moveusers.py:1]: u penis

Comment (benjaminl4917) [reply moveusers.py:1]: stole my shit

Comment (stephenlee91) [reply moveusers.py:1]: mmmm very good

**Review by TimothyYLin — CHANGES_REQUESTED** [review]

**Review by TimothyYLin — APPROVED** [review]

**Review by benjaminl4917 — APPROVED** [review]

**Pull request merged**